docs/user/attestation-provider.md

```diff
@@ -0,0 +1,113 @@
+# Becoming an Attestation Provider
+
+This guide is a walk through to become an [attestation provider](#attestation-provider).
+There are two situations in which it makes sense to do so:
+
+* Running local attestation providers, without any capital requirement, to act as a safeguard against malicious forks ;
+* Running an incentivized attestation provider to reap rewards, which contributes to the network and helps improve the decentralization of the network and provides attestations for the [state connector](#state-connector).
+
+??? note "Rewards"
+
+    Currently, the process of rewarding public attestation providers is still being discussed.
+
+## Prerequisites
+
+Before proceeding with the [installation](#installation), it is essential to follow every prerequisite.
+
+### System Requirements
+
+The following specifications were observed to be able to run all nodes on a single machine.
+Ideally, a single node should be run on each system in production for better performance and uptime.
```

Comment:
```suggestion
Ideally, a node should be run on each connected blockchain in production for better performance and uptime.
```

Reply:
Updated with the following snippet in [`a45798b` (#14)](https://github.com/flare-foundation/docs/pull/14/commits/a45798b04b3781889e74327f15aec12634ecb536)

```diff
-Ideally, a node should be run on each connected blockchain in production for better performance and uptime.
+Ideally, a dedicated node should be run for each connected blockchain in production for better performance and uptime.
```

WDYT?